pkg/apis/ark/v1/restic_repository.go

```diff
@@ -22,12 +22,20 @@ import (
 
 // ResticRepositorySpec is the specification for a ResticRepository.
 type ResticRepositorySpec struct {
-	// MaintenanceFrequency is how often maintenance should be run.
-	MaintenanceFrequency metav1.Duration `json:"maintenanceFrequency"`
+	// ForNamespace is the namespace this restic repository contains
+	// pod volume backups for.
+	ForNamespace string `json:"forNamespace"`
```

Comment:
Can we call this Namespace instead?

Reply:
I don't like `ForNamespace` or `Namespace` :) Maybe `PodNamespace`, `BackupNamespace`, something like that?

Reply:
What are some of the reasons you dislike Namespace?

Reply:
it's just very generic.  namespace of what?

Reply:
`TargetNamespace`? idk. naming is hard.